static/js/popovers.js

```diff
@@ -744,7 +744,22 @@ exports.register_click_handlers = function () {
     });
 
     $('body').on('click', '.respond_button', function (e) {
+        var textarea = $("#new_message_content");
+        var msgid = $(e.currentTarget).data("msgid");
+
         compose.respond_to_message({trigger: 'popover respond'});
+        channel.get({
+            url: '/json/messages/' + msgid,
+            idempotent: true,
+            success: function (data) {
+                if (textarea.val() === "") {
+                    textarea.val("```quote\n" + data.raw_content +"\n```\n");
+                } else {
+                    textarea.val(textarea.val() + "\n```quote\n" + data.raw_content +"\n```\n");
```

Comment:
@timabbott I think this is just work if we do `Quote and reply` on the same topics, so if we do that on the message that have different topic the original `textarea` content is lost..